lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3460,25 +3460,39 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
-                  , ( "any", [json|{ "any": [ "cosigner#0" ] }|] )
-                  , ( "some", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0" ]} }|] )
-                  , ( "all, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "some, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_until": 57297561 } ]} }|] )
-                  , ( "all, active_from 10", [json|{ "all": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "any, active_from 10", [json|{ "any": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "some, active_from 10", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 } ]} }|] )
-                  , ( "all, active_from 10, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_from 10, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
-                  , ( "some, active_from 10, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 }, { "active_until": 57297561 } ]} }|] )
+                  [ ( "all", [json|{ "all": [ "cosigner#0" ] }|], False )
+                  , ( "any", [json|{ "any": [ "cosigner#0" ] }|], False )
+                  , ( "some", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0" ]} }|], False )
+                  , ( "all, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_until": 57297561 } ] }|], False )
+                  , ( "any, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 } ] }|], False )
+                  , ( "some, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_until": 57297561 } ]} }|], False )
+                  , ( "all, active_from 10", [json|{ "all": [ "cosigner#0",  { "active_from": 10 } ] }|], True )
+                  , ( "any, active_from 10", [json|{ "any": [ "cosigner#0",  { "active_from": 10 } ] }|], True )
+                  , ( "some, active_from 10", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 } ]} }|], True )
+                  , ( "all, active_from 10, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] , True)
+                  , ( "any, active_until 57297561, active_from 58297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 }, { "active_from": 58297561 } ] }|] , False)
+                  , ( "some, active_from 10, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 }, { "active_until": 57297561 } ]} }|], False )
```

Comment:
I don't understand why this one, i.e.:
```
{
   "some":{
      "at_least":1,
      "from":[
         "cosigner#0",
         {
            "active_from":10
         },
         {
            "active_until":57297561
         }
      ]
   }
}
```
 doesn't need additional `validity_interval` constraint:
```
{
   "validity_interval":{
      "invalid_before":{
         "quantity":11,
         "unit":"slot"
      }
   }
}
```

And this one needs one:
```
{
   "all":[
      "cosigner#0",
      {
         "active_from":10
      },
      {
         "active_until":57297561
      }
   ]
}
```

Reply:
well, in case of at_least 1 we have (in absence of validity interval setting)
`(0, 57297561) OR (10, maxBound)` and validity (0, 32000) so the first interval encapsulate it, second not. But we have to satisfy at least 1 or 0 here

In case of `all`  we have to satisfy all conds,so we have timelock `(10, 57297561)` but default validity interval is `(0, 32000)` so it is NOT inside timelock. Hence we have to set validity to for example `(11, 32000)` to make it happen. 

I think it is correct behaviour